### PR TITLE
fixed validation messages

### PR DESCRIPTION
**Pull request by piotr-major-rmtcfm-com**

### CODE REVIEW

File: app/test/controllers/Todo/destroy.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ describe('Todos', () => {
 
             expect(body).to.have.property('errors');
             expect(body.errors).to.deep.include({
-                message: 'must be valid UUID',
+                message: 'Must be valid UUID.',
```
Comment (piotr-sko-prpl) [inline app/test/controllers/Todo/destroy.js]:
```suggestion
                message: 'Must be a valid UUID.',
```

File: app/test/controllers/Todo/show.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ describe('Todos', () => {
 
             expect(body).to.have.property('errors');
             expect(body.errors).to.deep.include({
-                message: 'must be valid UUID',
+                message: 'Must be valid UUID.',
```
Comment (piotr-sko-prpl) [inline app/test/controllers/Todo/show.js]:
```suggestion
                message: 'Must be a valid UUID.',
```

File: app/test/controllers/Todo/update.js [hunk 3]
```diff
@@ -117,7 +117,7 @@ describe('Todos', () => {
 
             expect(body).to.have.property('errors');
             expect(body.errors).to.deep.include({
-                message: 'must be valid UUID',
+                message: 'Must be valid UUID.',
```
Comment (piotr-sko-prpl) [inline app/test/controllers/Todo/update.js]:
```suggestion
                message: 'Must be valid a UUID.',
```

File: app/test/controllers/Users/destroy.js [hunk 4]
```diff
@@ -36,7 +36,7 @@ describe('Users', () => {
 
             expect(body).to.have.property('errors');
             expect(body.errors).to.deep.include({
-                message: 'must be valid UUID',
+                message: 'Must be valid UUID.',
```
Comment (piotr-sko-prpl) [inline app/test/controllers/Users/destroy.js]:
```suggestion
                message: 'Must be a valid UUID.',
```

File: app/test/controllers/Users/store.js [hunk 5]
```diff
@@ -116,7 +116,7 @@ describe('Users', () => {
             expect(body).to.have.property('errors');
             expect(body.errors).to.deep.include({
                 param: 'password',
-                message: 'password should be longer  than 8 characters'
+                message: 'Password should be longer  than 8 characters.'
```
Comment (piotr-sko-prpl) [inline app/test/controllers/Users/store.js]:
```suggestion
                message: 'Password should be longer than 8 characters.'
```

File: app/test/controllers/Users/update.js [hunk 6]
```diff
@@ -59,39 +59,39 @@ describe('Users', () => {
 
             expect(body).to.have.property('errors');
             expect(body.errors).to.deep.include({
-                message: 'must be valid UUID',
+                message: 'Must be valid UUID.',
```
Comment (piotr-sko-prpl) [inline app/test/controllers/Users/update.js]:
```suggestion
                message: 'Must be a valid UUID.',
```

File: app/validators/paginationValidator.js [hunk 7]
```diff
@@ -4,9 +4,9 @@ module.exports = [
     query('page')
         .optional()
         .custom(page => !isNaN(page))
-        .withMessage('Should be integer!'),
+        .withMessage('Should be integer.'),
```
Comment (piotr-sko-prpl) [inline app/validators/paginationValidator.js]:
```suggestion
        .withMessage('Should be an integer.'),
```

File: app/validators/paginationValidator.js [hunk 8]
```diff
@@ -4,9 +4,9 @@ module.exports = [
...
     query('perPage')
         .optional()
         .custom(perPage => !isNaN(perPage))
-        .withMessage('Should be integer!')
+        .withMessage('Should be integer.')
```
Comment (piotr-sko-prpl) [inline app/validators/paginationValidator.js]:
```suggestion
        .withMessage('Should be an integer.')
```

File: app/validators/userValidator.js [hunk 9]
```diff
@@ -6,50 +6,50 @@ async function isMailTaken(email, di, userId) {
...
 
 const withPassword = [
     body('password')
         .not()
         .isEmpty()
-        .withMessage('cannot be blank')
+        .withMessage('Password is required.')
         .isLength({ min: 8 })
-        .withMessage('password should be longer  than 8 characters')
+        .withMessage('Password should be longer  than 8 characters.')
```
Comment (piotr-sko-prpl) [inline app/validators/userValidator.js]:
```suggestion
        .withMessage('Password should be longer than 8 characters.')
```

File: app/validators/uuidValidator.js [hunk 10]
```diff
@@ -3,7 +3,7 @@ const { validate } = require('uuid');
 
 async function isUUIDValid(uuid) {
     if (!validate(uuid)) {
-        return Promise.reject('must be valid UUID');
+        return Promise.reject('Must be valid UUID.');
```
Comment (piotr-skowronski-prpl) [inline app/validators/uuidValidator.js:6]:
```suggestion
        return Promise.reject('Must be a valid UUID.');
```

File: app/validators/uuidValidator.js [hunk 11]
```diff
@@ -3,7 +3,7 @@ const { validate } = require('uuid');
 
 async function isUUIDValid(uuid) {
```
Comment (piotr-skowronski-prpl) [inline app/validators/uuidValidator.js:4]:
```suggestion
function isUUIDValid(uuid) {
```

File: app/validators/uuidValidator.js (hunk 10 above)
Comment (piotr-sko-prpl) [inline app/validators/uuidValidator.js]:
```suggestion
        return Promise.reject('Must be a valid UUID.');
```

and remove `async`

**Pull request merged**